data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1789 @@
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Lunarium Initial Log"
+	invisible
+	landing
+	to offer
+		has "Lunarium introduced"
+	on offer
+		log "Factions" "Lunarium Redoubt" "The Lunarium Redoubt is a resistance movement, opposed to the Heliarchs, within the Coalition, aiming to free it from their rule.
+		fail
+
+mission "Resistance: Smuggling: Charity 1"
+	name "Charity Delivery"
+	description "Deliver a care package of <cargo> to <destination>, as part of a charity initiative."
+	minor
+	cargo "charity supplies" 22
+	source
+		near "3 Spring Rising" 2 6
+		not government "Heliarch"
+	destination "Fourth Shadow"
+	to offer
+		random < 40
+		has "Coalition: First Contact: done"
+		not "Heliarch License: done"
+		not "Heliarch Investigation 2: active"
+		not "Resistance: Questions: active"
+	on offer
+		conversation
+			branch metchiree
+				has "Resistance: Questions: done"
+			`A Kimek in a blue uniform approaches you. "Good day, Captain. Interested in helping with some charity work, would you be?" She asks.`
+			choice
+				`	"What sort of charity work?"`
+				`	"Sorry, I'm busy right now."`
+					decline
+			`	"Very prosperous, many Kimek worlds have become, but still lagging behind, a few of our planets are," she explains. "Very poor, the people of <planet> are, and many amenities such as computers and communicators, they lack. To help them, our more fortunate want, and offered up many supplies, they have. In need of captains to transport them to <planet>, we are," she finishes, looking at your ship.`
+			choice
+				`	"I'd be glad to help you. Have the supplies loaded into my ship."`
+				`	"That's a noble cause, but I don't work as a cargo hauler, sorry."`
+					decline
+			`	Her mouth produces something of a chirping, clicking sound, "Grateful I am, Captain. Pay you much, we cannot, but helping many, you will be. Of that you I assure."`
+			`	She signals to some workers with similar uniforms, and in a few minutes they bring the cargo crates to your ship. More colleagues are already on <planet> working with more captains, she explains, and suggests to look for them when you arrive. She thanks you again, and wishes you a good trip to <planet>.`
+				accept
+			label metchiree
+			`A Kimek in a blue uniform approaches you, and you recognize her as Chiree, the Lunarium leader you met when joining them. She asks to speak in private, and you two go inside your ship. "Much better," she says once the hatch is closed. "Easier to explain you the job, it will be, if speak freely and truly of it, I can.`
```

Comment:
```suggestion
			`A Kimek in a blue uniform approaches you, and you recognize her as Chiree, the Lunarium leader you met when joining them. She asks to speak in private, and you two go inside your ship. "Much better," she says once the hatch is closed. "Easier to explain to you the job, it will be, if speak freely and truly of it, I can.`
```